model_compression_toolkit/gptq/pytorch/gptq_training.py

```diff
@@ -105,25 +106,53 @@ def _is_gptq_applicable(self,
                          f"without a kernel isn't supported.")
         return node.is_weights_quantization_enabled()
 
-    def gptq_wrapper(self, n: BaseNode, layer: Module) -> Union[qi.PytorchQuantizationWrapper, Module]:
+    def gptq_wrapper(self,
+                     n: BaseNode,
+                     layer: Module,
+                     include_activation_quantizers: bool = False) -> Union[qi.PytorchQuantizationWrapper, Module]:
         """
         A function which takes a computational graph node and a pytorch layer and perform the quantization wrapping.
 
         Args:
             n: A node of mct graph.
             layer: A pytorch layer
+            include_activation_quantizers: Whether or not to use the wrapper for the activation quantizer
 
         Returns: Wrapped layer if the layer should be wrap, otherwise returns the layer as is.
         """
 
         if self._is_gptq_applicable(n):
             weights_quantizers, activation_quantizers = quantization_builder(n, self.gptq_config)
-            return qi.PytorchQuantizationWrapper(layer,
-                                                 weights_quantizers=weights_quantizers,
-                                                 activation_quantizers=activation_quantizers)
+            if include_activation_quantizers:
+                return qi.PytorchQuantizationWrapper(layer,
+                                                     weights_quantizers=weights_quantizers,
+                                                     activation_quantizers=activation_quantizers)
+            else:
+                return qi.PytorchQuantizationWrapper(layer,
+                                                     weights_quantizers=weights_quantizers)
         else:
             return layer
 
+    def get_activation_quantizer_holder(self, n: BaseNode) -> Union[None, Callable]:
+        """
+        Retrieve a PytorchActivationQuantizationHolder layer to use for activation quantization of a node.
+        If the layer is not supposed to be wrapped with an activation quantizer - return None.
+        Args:
+            n: Node to attach a PytorchActivationQuantizationHolder to its output.
+        Returns:
+            A PytorchActivationQuantizationHolder module for the node's activation quantization.
+        """
+        if self._is_gptq_applicable(n): # If it was wrapped before - add holder instead
+            _, activation_quantizers = quantization_builder(n, self.gptq_config)
```

Comment:
It is a good question, but it is not related to this PR. Maybe @ofirgo can answer?

Reply:
I'm not sure which function are you referring to.
Note that this entire method needs to be changed, there is a bug here that is currently being fixed by @reuvenperetz in another PR, so the method here should match the equivalent method in Keras GPTQ

Reply:
Please see what I did in the current open PR (skip the applicable check, and the None return option. Also, the model builder uses this function only if activation quantization is enabled for the node).

About their location, @eladc-git , this is because the wrapper method of QAT was in facade (which seems wrong) and this function should not be in the facade as well. I think it's better in a different PR to move QAT wrapper function to the builder file (and see if it's possible to do the same for GPTQ, because it's a property of the GPTQTrainer object)